src/app/pages/salidas/tabla-entrada-aprobada/tabla-entrada-aprobada.component.html

```diff
@@ -376,12 +429,14 @@
     <mat-expansion-panel [expanded]="true" hideToggle>
       <mat-expansion-panel-header>{{'GLOBAL.salidas.elementos' | translate}}
       </mat-expansion-panel-header>
-      <ngx-tabla-elementos-asignados [entradaId]="consecutivoEntrada" [actaRecibidoId]="actaRecibidoId" *ngIf="detalle">
+      <ngx-tabla-elementos-asignados [entradaId]="consecutivoEntrada"
+        [edicionSalida]="edicion" [salida_id]="salida_id"
+	    [actaRecibidoId]="actaRecibidoId" *ngIf="detalle">
       </ngx-tabla-elementos-asignados>
     </mat-expansion-panel>
   </mat-accordion>
 </mat-card>
 <mat-card-actions>
   <button mat-raised-button color="secondary" (click)="onVolver()" class="float-left text-uppercase"
-    *ngIf="detalle">{{'GLOBAL.volver' | translate}}</button>
+    *ngIf="detalle && !edicion">{{'GLOBAL.volver' | translate}}</button>
 </mat-card-actions>
```

Comment:
Dejar la última línea en blanco, sin espacios ni nada, para archivos de texto plano
```suggestion
</mat-card-actions>

```